src/authz/stores/typeorm/entities/authorization-code.entity.ts

```diff
@@ -33,6 +35,7 @@ export class AuthorizationCodeEntity {
   @Column({ nullable: true })
   used_at?: Date;
 
+  // Highly sensitive: Access token - can be encrypted if transformer is provided
   @Column()
   github_access_token: string;
```

Comment:
This doesn't have to be in the database at all. For sure it is a mistake on my end, and it is safe to remove this.